source/features/one-click-review-submission.tsx

```diff
@@ -22,11 +24,6 @@ function addButtons(radios: HTMLInputElement[]): void {
 		);
 	}
 
-	// "Comment" button must be first
-	if (radios.length > 1) {
-		radios.push(radios.shift()!);
-	}
```

Comment:
Very long in the tooth. Long story:

- https://github.com/refined-github/refined-github/pull/5054#issuecomment-963575566